cypress/e2e/po/components/ember/ember-input.po.ts

```diff
@@ -6,12 +6,12 @@ export default class EmberInputPo extends EmberComponentPo {
    * @param value Value to be typed
    * @returns
    */
-  set(value: string): Cypress.Chainable {
-    this.input().should('be.visible');
-    this.input().focus();
-    this.input().clear();
+  set(value: string, index = 0): Cypress.Chainable {
```

Comment:
The `ctor` approach would work by something like below, as input just looks at self

```
new EmberInputPo(<selector to input field>)
```

Reply:
@richard-cox I don't think the `ctor` approach works in this case since the input fields do not have unique identifiers (other than an ember id (e.g. id="ember175") which is inconsistent between forms so that can't be used here). For this approach to work, we would need to add a `data-testid` to these fields however I’m unfamiliar with ember code. could you help me with this?

Reply:
In this context selectors are css selectors and don't need to contain `data-testid`. For instance it could be `[data-testid="<something parent like?"] input:nth-of-type(1)`

Reply:
Unfortunately, using `form input:nth-of-type(1)` targets all 4 inputs 

<img width="1130" alt="Screenshot 2024-01-19 at 12 22 38 PM" src="https://github.com/rancher/dashboard/assets/127343932/30ee6d2a-fabc-4aff-a406-069cb2339a84">

Reply:
That was just an example. You would need to tweak the selector to be more specific. I'm guessing there are other forms on the page with input elements. So maybe something like `form.model-container`

Reply:
I've tried all sorts of selector combinations. :nth-of-type(1) still targets all 4 inputs on the modal

Reply:
`.modal-open .inline-form:nth-of-type(2) input` will do the trick

Reply:
that did do the trick, thank you @richard-cox! updated in https://github.com/rancher/dashboard/pull/10161/commits/e093c5c62425893677980aaa3cb60e30b0946f49